packages/messaging/src/helpers/registerDefaultSw.ts

```diff
@@ -24,8 +27,10 @@ export async function registerDefaultSw(
   messaging: MessagingService
 ): Promise<void> {
   try {
-    messaging.swRegistration = await navigator.serviceWorker.register(
-      DEFAULT_SW_PATH,
+    const container = navigator.serviceWorker;
+    messaging.swRegistration = await safeServiceWorkerContainer.register(
+      container,
+      trustedResourceUrl`/firebase-messaging-sw.js`,
```

Comment:
This is used in `getToken` and `deleteToken`.

The tests are passing. Is that enough to be confident that this works?